src/Lean/Parser/Term.lean

```diff
@@ -251,6 +278,12 @@ and solved by typeclass inference of the specified class.
 -/
 def instBinder := ppGroup <| leading_parser
   "[" >> withoutPosition (optIdent >> termParser) >> "]"
+/-- A `bracketedBinder` matches any kind of binder group with an explicit bracketing syntax:
```

Comment:
@digama0 

In that case, what do you think of this wording?

```suggestion
/-- A `bracketedBinder` matches any kind of binder group that uses brackets:
```

Reply:
@digama0 LGTM! However, I am little surprised @semorrison (an Australian) agreed with that default because, according to my knowledge and the [Wikipedia page](https://en.wikipedia.org/wiki/Bracket), defaulting the general word bracket to square brackets (i.e., `[]`) is only standard in the US.